### ISSUE TITLE
pasa raised an error on DB_connect during training

### ISSUE THREAD

**Issue by iwangtoknow** [nextgenusfs/funannotate#247]
**Are you using the latest release?**
If you are not using the latest release of funannotate, please upgrade, if bug persists then report here.
I'm using docker.
**Describe the bug**
SQlite connect error during PASA train, but only raised in certainly.
**What command did you issue?**
Copy/paste the command used.
`funannotate-train.py -i nuclear.masked.sorted.fasta -o 20181224 -l left.fq -r right.fq --stranded no --pacbio_isoseq isoseq.fastq --jaccard_clip --species Aspergillus nidulans --strain BJCH_M5 --cpus 36`
**Logfiles**
Please provide relavent log files of the error. 
```bash
[12:25 AM]: OS: linux2, 72 cores, ~ 66 GB RAM. Python: 2.7.14
[12:25 AM]: Running funannotate v1.5.2-26810fb
[12:25 AM]: Existing Trinity results found: 20181224/training/trinity.fasta
[12:25 AM]: Existing SeqClean output found: 20181224/training/20181224/training/trinity.fasta.clean
[12:25 AM]: Existing BAM alignments found: 20181224/training/trinity.alignments.bam, 20181224/training/transcript.alignments.bam
[12:25 AM]: Running PASA alignment step using 26,642 transcripts
Traceback (most recent call last):
  File "/home/linuxbrew/funannotate/bin/funannotate-train.py", line 967, in <module>
    runPASAtrain(genome, trinity_transcripts, cleanTranscripts, os.path.abspath(trinityGFF3), stringtieGTF, args.stranded, args.max_intronlen, args.cpus, organism_name, PASA_tmp)
  File "/home/linuxbrew/funannotate/bin/funannotate-train.py", line 405, in runPASAtrain
    with open(os.path.join(folder, pasaDBname+'.pasa_assemblies_described.txt'), 'rU') as description:
IOError: [Errno 2] No such file or directory: '20181224/training/pasa/Aspergillus_nidulans_BJCH_M5.pasa_assemblies_described.txt'
```
```bash
Committed 1000 PASA assemblies       
Committed 2000 PASA assemblies       
Committed 3000 PASA assemblies       
[1/8] scaffolds processed.           
Committed 4000 PASA assemblies       
Committed 5000 PASA assemblies       
Committed 6000 PASA assemblies       
[2/8] scaffolds processed.           
Committed 7000 PASA assemblies       
Committed 8000 PASA assemblies       
Committed 9000 PASA assemblies       
Committed 10000 PASA assemblies       
[3/8] scaffolds processed.           
Committed 11000 PASA assemblies       
Committed 12000 PASA assemblies       
[4/8] scaffolds processed.           
Committed 13000 PASA assemblies       
Committed 14000 PASA assemblies       
Committed 15000 PASA assemblies       
Committed 16000 PASA assemblies       
[5/8] scaffolds processed.           DBD::SQLite::db do failed: UNIQUE constraint failed: asmbl_link.asmbl_acc, asmbl_link.cdna_acc at /home/linuxbrew/PASApipeline/PerlLib/DB_connect.pm line 221.
failed query: <insert into asmbl_link (asmbl_acc, cdna_acc) values (?,?)>	values: asmbl_16999 f1p1
Errors: UNIQUE constraint failed: asmbl_link.asmbl_acc, asmbl_link.cdna_acc
 at /home/linuxbrew/PASApipeline/PerlLib/DB_connect.pm line 233.
	DB_connect::RunMod(DB_connect=HASH(0xbc6eb8), "insert into asmbl_link (asmbl_acc, cdna_acc) values (?,?)", "asmbl_16999", "f1p1") called at /home/linuxbrew/PASApipeline/scripts/assembly_db_loader.dbi line 153
Issuing rollback() due to DESTROY without explicit disconnect() of DBD::SQLite::db handle database=/home/linuxbrew/data/20181224/training/pasa/Aspergillus_nidulans_BJCH_M5;host=localhost.
Error, cmd: /home/linuxbrew/PASApipeline/scripts/assembly_db_loader.dbi -M '/home/linuxbrew/data/20181224/training/pasa/Aspergillus_nidulans_BJCH_M5' > pasa_run.log.dir/alignment_assembly_loading.out died with ret 6400 No such file or directory at /home/linuxbrew/PASApipeline/PerlLib/Pipeliner.pm line 186.
	Pipeliner::run(Pipeliner=HASH(0x23227c8)) called at /home/linuxbrew/PASApipeline/Launch_PASA_pipeline.pl line 1044
```
**OS/Install Information**
 -  output of `funannotate check --show-versions`

Should I re-build the docker?

**WANG**

**Comment by iwangtoknow** [comment]
Hi Jon,
I think the problem is `Errors: UNIQUE constraint failed: asmbl_link.asmbl_acc, asmbl_link.cdna_acc`
The `asmbl_link.cdna_acc` is not UNIQUE
I found an old issue in pasa github [https://github.com/PASApipeline/PASApipeline/issues/32](url)
I'm re-runing with '/' changed by '_' in my isoseq.fastq
I'll keep on the process.

**WANG**

**Comment by iwangtoknow** [comment]
Hi Jon,
Seems like this error has been found.
Now I provide full length non chimeric reads for RNA evidence.
The isoforms output from SMRT portal isoseq.1 protocol named with '/'  could cause unique error.

**WANG**

**Comment by iwangtoknow** [comment]
I'll do it, thanks Jon.
Another word, even I changed all the '/' in the description line, there still could have reads with the same name.
My isoseq data is produced by SMRT Portal isoseq.1 protocol polished. I combined 3 cells data both high-quality and low-qulity final isoforms.

Another kind of full-length transcripts output from isoseq is flnc reads in fasta format. Also have the same problem.

After I modified the '/' with '_', I check the duplicated name with `sort | uniq -c | awk '$1!=1'`

Thanks Jon.

**WANG**
